project-base/src/Shopsys/ShopBundle/Resources/config/services.yml

```diff
@@ -53,6 +53,8 @@ services:
 
     Shopsys\FrameworkBundle\Model\Product\ProductDataFactoryInterface: '@Shopsys\ShopBundle\Model\Product\ProductDataFactory'
 
+    Shopsys\FrameworkBundle\Model\Product\ProductOnCurrentDomainFacadeInterface: '@Shopsys\FrameworkBundle\Model\Product\ProductOnCurrentDomainElasticFacade'
```

Comment:
This configuration should be part of framework bundle for non-BC change (but still over-writable by project-base)

Reply:
Actually, usage of this facade is only in project-base and if you don't update controllers you have perfectly working application. 
But in the future, it should be in FW though.

Reply:
you are right, good point with controllers :+1: